modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowNotifications.testcase

```diff
@@ -112,12 +112,48 @@ definition {
 	}
 
 	@description = "Verify that the user can open new user details by notifications"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanOpenNewUserDetailsByNotifications {
+		property test.name.skip.portal.instance = "WorkflowNotifications#CanOpenNewUserDetailsByNotifications";
 
-		// TODO LRQA-70881 CanOpenNewUserDetailsByNotifications pending implementation
+		task ("Configure Workflow to Single Approver for User") {
+			Workflow.openWorkflowListView();
 
+			Navigator.gotoNavItem(navItem = "Configuration");
+
+			Workflow.configureWorkflow(
+				workflowDefinition = "Single Approver",
+				workflowResourceValue = "User");
+		}
+
+		task ("Disable Captcha to Create a New User") {
+			WorkflowUser.disableCaptchaToCreateNewUserAndSendPassword();
+		}
+
+		task ("Create a New User") {
+			User.logoutPG();
+
+			WorkflowAsset.createNewUserAccount(
+				newPassword = "password",
+				userEmailAddress = "user@liferay.com",
+				userFirstName = "userfn",
+				userLastName = "userln",
+				userScreenName = "usersn");
+		}
```

Comment:
can you combine these two tasks into a unique one? For example: task("When create a new User")